pom.xml

```diff
@@ -163,6 +163,16 @@ under the License.
 			<type>jar</type>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
```

Comment:
I don't think it is necessary to add these dependency to the root pom. We added a dedicated Maven module for micro benchmarks (`flink-benchmark`). This is where the code and the dependencies should go, IMO.